cmd/pinniped-proxy/src/cli.rs

```diff
@@ -0,0 +1,19 @@
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
```

Comment:
This is an example of invoking a macro - in this case, we're asking the Rust compiler to automatically derive functionality to be able to print debug output of the struct below, as well as derive functionality for the struct below to be used as the structured options for a command-line CLI.